pkg/platform/helper.go

```diff
@@ -1112,7 +1112,7 @@ func WaitForVirtualClusterInstance(ctx context.Context, managementClient kube.In
 
 	if virtualClusterInstance.Status.Phase == storagev1.InstanceSleeping {
 		log.Info("Wait until vcluster wakes up")
-		defer log.Donef("Successfully woken up vcluster %s", name)
+		defer log.Donef("vCluster %s wakeup successful", name)
```

Comment:
didn't we say that it should either be `vCluster instance` or `virtual cluster`?